### DISCUSSION TITLE
Security scan for CVE

### DISCUSSION THREAD

**Discussion by BrinaD1** [microsoft-search/pnp-modern-search#1817] [Q&A]
Hello, I've had a request from the security team to scan this code to see if it exposes any critical [CVEs](https://en.wikipedia.org/wiki/Common_Vulnerabilities_and_Exposures) before we can install it in our tenant. Does anyone have recommendations for a tool that could do this? 

I also saw that Github [provides scanning for vulnerabilities](https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors) but that it hasn't been done in this repo. Has anyone used this functionality before and found it useful?

TIA!

**Comment by wobba** [comment]
Not sure of one. We do have some issues which we will address listed at https://github.com/microsoft-search/pnp-modern-search/issues/1803
